core/src/main/java/org/infinispan/configuration/cache/ClusteringConfiguration.java

```diff
@@ -19,7 +19,7 @@
          AttributeDefinition.builder("remoteTimeout", TimeUnit.SECONDS.toMillis(15)).build();
 
    static AttributeSet attributeDefinitionSet() {
-      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE);
+      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE, REMOTE_TIMEOUT);
```

Comment:
this is confusing. REMOTE_TIMEOUT exists in the SyncConfiguration. I'm not sure which one is really used or if it is really a good idea to add it here...